test/Hyperwallet.spec.js

```diff
@@ -247,17 +247,20 @@ describe("Hyperwallet", () => {
         /** @test {Hyperwallet#listUsers} */
         it("should do get call with options", () => {
             const callback = () => null;
-            client.listUsers({ test: "value" }, callback);
-
+            client.listUsers({ clientUserId: "test-client-user-id", status: "test-status" }, callback);
             apiClientSpy.should.have.been.calledOnce();
-            apiClientSpy.should.have.been.calledWith("users", { test: "value" });
+            apiClientSpy.should.have.been.calledWith("users", { clientUserId: "test-client-user-id", status: "test-status" });
         });
 
+        /** @test {Hyperwallet#listUsers} */
```

Comment:
pls add blank lines between the it tests

Reply:
done